src/libutil/util.cc

```diff
@@ -1,15 +1,23 @@
 #include "util.hh"
 #include "fmt.hh"
+#include "sync.hh"
+#include "finally.hh"
+#include "serialise.hh"
+#include "cgroup.hh"
+#include "hash.hh"
 
 #include <array>
 #include <cctype>
 #include <iostream>
 #include <grp.h>
 #include <regex>
 
+#include <sodium.h>
 
 namespace nix {
 
+const std::string nixVersion = PACKAGE_VERSION;
+
```

Comment:
Is this used?

Reply:
Well, yes, the symbol needs to be defined in that library? It's used for `set_user_agent`.